x-pack/plugins/elastic_assistant/server/services/app_context.test.ts

```diff
@@ -160,5 +160,19 @@ describe('AppContextService', () => {
         expect.objectContaining(assistantFeatures)
       );
     });
+
+    it('allows registering a subset of all available features', () => {
+      const pluginName = 'pluginName';
+      const featuresSubset: Partial<AssistantFeatures> = {
+        assistantModelEvaluation: true,
+      };
+
+      appContextService.start(mockAppContext);
+      appContextService.registerFeatures(pluginName, featuresSubset);
+
+      expect(appContextService.getRegisteredFeatures(pluginName)).toEqual(
+        expect.objectContaining({ ...assistantFeatures, ...featuresSubset })
+      );
+    });
```

Comment:
@andrew-goldstein -- fix for partial registration added in [6f17d5e](https://github.com/elastic/kibana/pull/174317/commits/6f17d5e4de092dfc2f4322249d76bf21d4a70e2b). Here's the test to ensure it's functionality.